manifests/apt.pp

```diff
@@ -13,31 +13,19 @@
     }
   }
 
-  package { $::duo_unix::duo_package:
-    ensure  => $package_state,
-    require => [
-      File[$repo_file],
-      Exec['Duo Security GPG Import'],
-      Exec['duo-security-apt-update']
-    ]
-  }
-
-  file { $repo_file:
-    owner   => 'root',
-    group   => 'root',
-    mode    => '0644',
-    content => "deb ${repo_uri}/${facts['operatingsystem']} ${facts['lsbdistcodename']} main",
-    notify  => Exec['duo-security-apt-update']
-  }
-
-  exec { 'duo-security-apt-update':
-    command     => '/usr/bin/apt-get update',
-    refreshonly => true
-  }
+  if $::duo_unix::manage_repo {
+    apt::source { 'duosecurity':
+      location => $repo_uri,
+      repos    => 'main',
+      key      => {
+        'id'     => '08C2A645DDF240B85844068D7A450864C1A07A85',
```

Comment:
"future-me who is going to stumble over this after the cert expired and we're rushing to get a new release pushed" would love the command to calculate/verify that fingerprint. :)

Reply:
This fingerprint was just me importing the key into gpg and running `gpg --list-fingerprints`. There might be a better way to do this using `apt-key`, but I'm not familiar enough with the tool.